docs/source/usage/solver.rst

```diff
@@ -42,6 +42,8 @@ The first way to add a repository is from a list of |PackageInfo| using
 
    import libmambapy
 
+   mamba_context = libmambapy.Context()
```

Comment:
I'm actually wondering if this is necessary since the tests in `libmambapy` are working without adding this (if I'm not mistaken)?

Reply:
Indeed, `libmamba.solver`, `libmamba.utils` and `libmamba.specs` work independently from the Context.

Reply:
Alright ,but then there is no example where I should add the line?

Reply:
Hmm no I think it's actually independent from the logging output. I didn't find any reference to it in the corresponding folders in `libmamba`.